ts/api/api.ts

```diff
@@ -362,3 +362,21 @@ class InitializeOnDemandApi implements OnDemandOpenYoloApi {
 }
 
 export const openyolo: OnDemandOpenYoloApi = new InitializeOnDemandApi();
+
+// Export the public methods.
+const windowAsAny = window as any;
+windowAsAny['openyolo'] = openyolo;
+InitializeOnDemandApi.prototype['setProviderUrlBase'] =
```

Comment:
For these, we found that as tsc preserves comments you could use a JSDoc comment with @export on each method that needs to be preserved. That would at least seem cleaner than these no-op redeclarations.